### ISSUE TITLE
[Bug] add notes about tctl

### ISSUE THREAD

**Issue by gedw99** [temporalio/ui#172]
I have getting 404 flash messages in the web gui until i gave it this 
```
./tctl --namespace default namespace register
```

maybe worth explaining this in the README for other newbies like me ?

**Comment by feedmeapples** [comment]
@gedw99 thanks for filing!
moved it under the UI repo for addressing

**Comment by stevekinney** [comment]
Awesome, thanks! We have adding a number of classes of errors to the UI on our roadmap for the next 2 weeks. This is a super helpful callout that I didn't have on that list.

**Comment by stevekinney** [comment]
@feedmeapples You solved this #260, right?

**Comment by feedmeapples** [comment]
not exactly, we could show for example a message saying `run tctl --namespace <name> namespace register` next to the 404. @canvanooo thoughts?
